packages/cozy-client/src/flagship-certification/flagship-certification.js

```diff
@@ -82,12 +82,19 @@ export const certifyFlagship = async (certificationConfig, client) => {
     )
   }
 
-  const stackChallengeNonce = await getStackChallenge(client)
+  try {
+    const stackChallengeNonce = await getStackChallenge(client)
 
-  const appAttestation = await getAppAttestationFromStore(
-    stackChallengeNonce,
-    certificationConfig
-  )
+    const appAttestation = await getAppAttestationFromStore(
+      stackChallengeNonce,
+      certificationConfig
+    )
 
-  await giveAppAttestationToStack(appAttestation, stackChallengeNonce, client)
+    await giveAppAttestationToStack(appAttestation, stackChallengeNonce, client)
+  } catch (e) {
+    console.warn(
+      '[FLAGSHIP_CERTIFICATION] Certification failed but the cozy-stack should continue with 2FA certification'
```

Comment:
this sentence is not easy to understand. 

Negative "failed"  "but" "should"...

Reply:
I made the edit here: https://github.com/cozy/cozy-client/compare/d9ea74471e1e23ed356a9e710722a2939f4b6972..559190a18cb89ec0392b90d02f58b85f577e03e7